src/theme/variables.css

```diff
@@ -245,6 +245,10 @@ http://ionicframework.com/docs/theming/ */
   }
 }
 
+ion-item > ion-label {
+  white-space: normal !important;
```

Comment:
does this not work without important? why not?

Reply:
Yes it is not over riding the white space without important.
![label](https://user-images.githubusercontent.com/82817616/153051570-fd674e2e-3e56-44aa-837c-95a72eea5484.png)

Reply:
we need a better solution than this because if we use this then we won't be able to over write this with a utility class.